tmt/utils.py

```diff
@@ -4597,17 +4597,19 @@ def validate_git_status(test: 'tmt.base.Test') -> tuple[bool, str]:
 
 def generate_runs(
         path: Path,
-        id_: Optional[str] = None) -> Iterator[Path]:
+        id_: tuple[str]) -> Iterator[Path]:
     """ Generate absolute paths to runs from path """
     # Prepare absolute workdir path if --id was used
-    if id_:
-        run_path = Path(id_)
-        if '/' not in id_:
-            run_path = path / run_path
-        if run_path.is_absolute():
-            if run_path.exists():
+    run_path = None
+    for id_name in id_:
+        if id_name:
```

Comment:
Can `id_name` not be a valid value? `id_` is `tuple[str, ...]`, i.e. all items will be strings. Can any of them be empty? If so, would it be better to raise an error rather than ignore `--id ""`?